src/messages/GE.json

```diff
@@ -447,5 +447,6 @@
   "Settings": "Settings",
   "sign in other account": "Sign in with a different account",
   "Help": "Help",
-  "Documentation": "Documentation"
+  "Documentation": "Documentation",
+  "Graph Documentation": "Microsoft Graph API Reference"
```

Comment:
Actually, I don't think Documentation is used anywhere, you could rename that one instead. 
And could you add some localization for the "Get started with Graph Explorer" text. I must have forgotten.

Reply:
Nice catch @ElinorW :)